packages/origin-ui-core/src/components/AutoSupplyDeviceTable.tsx

```diff
@@ -80,19 +81,27 @@ export function AutoSupplyDeviceTable() {
         loadPage(1);
     }, [userOffchain, deviceClient]);
 
+    const currentYear = moment(new Date().valueOf());
+    const nextYear = moment(currentYear).add(1, 'years');
+
     const columns = [
         { id: 'type', label: 'Type' },
         { id: 'facility', label: 'Facility' },
         { id: 'price', label: 'Price' },
         { id: 'status', label: 'Status' },
-        { id: 'certified', label: 'To be certified for 2020/2021 (Mwh)' }
+        {
+            id: 'certified',
+            label: `To be certified for ${currentYear.format('YYYY')}/${nextYear.format(
```

Comment:
Please use localized strings like this: https://github.com/energywebfoundation/origin/blob/ce900daed490d632ebee2feb6a4f2e4074ba149d/packages/origin-ui-core/src/components/ProducingDeviceDetailView.tsx#L141

For all these columns above.